packages/cli/src/runDeploy.ts

```diff
@@ -43,9 +43,9 @@ export const deployOptions = {
     type: "string",
     desc: "The deployment salt to use. Defaults to a random salt.",
   },
-  awsKmsKeyId: {
-    type: "string",
-    desc: "Optional AWS KMS key ID. If set, the World is deployed using a KMS signer instead of local private key.",
+  aws: {
```

Comment:
since we call everything "kms" I wonder if this flag should be `--kms` or `--awsKms`

Reply:
agreed, I also like the specificity of KMS